### DISCUSSION TITLE
v4.2.1-rc.2

### DISCUSSION THREAD

**Discussion by prestonvanloon** [OffchainLabs/prysm#13530] [General]
Release candidate for v4.2.1, including:

- Fixes from Goerli Dencun fork
- Holesky and Sepolia Dencun fork epochs

Full notes to come in the final release.

Full diff https://github.com/prysmaticlabs/prysm/compare/v4.2.0...v4.2.1-rc.2

<hr /><em>This discussion was created from the release <a href='https://github.com/prysmaticlabs/prysm/releases/tag/v4.2.1-rc.2'>v4.2.1-rc.2</a>.</em>

**Comment by Brindrajsinh-Chauhan** [comment]
Would be grateful to know when can the release v4.2.1 be expected since the sepolia hardfork is approaching close.
Thank you

**Reply by prestonvanloon** [reply]
It's expected this week. You may run the release candidate for Sepolia in the meantime.
